modules/apps/web-experience/journal/journal-test/build.gradle

```diff
@@ -15,11 +15,11 @@ dependencies {
 	testIntegrationCompile group: "com.liferay", name: "com.liferay.dynamic.data.mapping.api", version: "3.0.0"
 	testIntegrationCompile group: "com.liferay", name: "com.liferay.dynamic.data.mapping.io", version: "2.0.0"
 	testIntegrationCompile group: "com.liferay", name: "com.liferay.dynamic.data.mapping.service", version: "2.0.0"
+	testIntegrationCompile group: "com.liferay", name: "com.liferay.dynamic.data.mapping.test.util", version: "2.0.0"
 	testIntegrationCompile group: "com.liferay", name: "com.liferay.journal.api", version: "2.0.0"
 	testIntegrationCompile group: "com.liferay", name: "com.liferay.journal.service", version: "2.0.0"
 	testIntegrationCompile group: "com.liferay", name: "com.liferay.journal.web", version: "1.0.0"
 	testIntegrationCompile group: "com.liferay", name: "com.liferay.registry.api", version: "1.0.0"
 	testIntegrationCompile group: "com.liferay.portal", name: "com.liferay.portal.impl", version: "2.0.0"
 	testIntegrationCompile group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "2.0.0"
-	testIntegrationCompile project(":apps:forms-and-workflow:dynamic-data-mapping:dynamic-data-mapping-test-util")
```

Comment:
New method introduced in superclass at `dynamic-data-mapping-test-util`

https://github.com/brianchandotcom/liferay-portal/pull/38872/commits/18cdb45ae193788310705750eca965cf1d960bfd#diff-8f18d1a14d21a3247f11c185327023e9R89